cli/dcoscli/data/help/node.txt

```diff
@@ -11,8 +11,9 @@ Usage:
     dcos node diagnostics delete <bundle>
     dcos node diagnostics download <bundle> [--location=<location>]
     dcos node list-components [--leader --mesos-id=<mesos-id> --json]
-    dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<agent-id>]
+    dcos node log [--follow --lines=N --leader --master --mesos-id=<mesos-id> --slave=<slave-id>]
                   [--component=<component-name> --filter=<filter>...]
+    dcos node metrics (--mesos-id=<mesos-id>) [--filter=<name>... | --json]
```

Comment:
I don't think we want `--mesos-id` in parens. (x | y) is mutual exclusion in docopt land